File: Programmers/Level1/음양더하기/음양더하기.py
def solution(absolutes, signs):
    answer = 0
    for i in range(len(absolutes)):
        answer =  answer + absolutes[i] if signs[i] else answer - absolutes[i]            
    return answer

# 다른 풀이
def solution(absolutes, signs):
    answer = 0
    for a, s in zip(absolutes, signs):
        answer =  answer + a if s else answer - a
    return answer

File: Programmers/Level1/음양더하기/음양더하기.js
function solution(absolutes, signs) {
    let answer = 0;
    for (let i = 0; i < absolutes.length; i++) {
        answer = signs[i] ? answer + absolutes[i] : answer - absolutes[i];
    }
    return answer;
}

// 다른 풀이
function solution(absolutes, signs) {
    return absolutes.reduce(
        (acc, val, i) => acc + val * (signs[i] ? 1 : -1),
        0
    );
}
